package.json

```diff
@@ -113,7 +114,7 @@
     "karma-jasmine": "~5.1.0",
     "karma-jasmine-html-reporter": "^2.0.0",
     "karma-mocha-reporter": "^2.2.5",
-    "lint-staged": "^15.2.0",
+    "lint-staged": "^14.0.1",
```

Comment:
is it intentional downgrade?

Reply:
I guess I will delete this files from commit, not intentional changes

Reply:
removed.